FitGoal/FitGoal/Singup&Singin/CustomUIViews/SocialMediaAuthenticationView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  SocialMediaButtonsView.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 20/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class SocialMediaAuthenticationView: UIView {
+    
+    private let dividerStack: UIStackView = {
+        let stack = UIStackView()
+        stack.axis = .horizontal
+        stack.alignment = .center
+        stack.distribution = .fill
+        stack.spacing = 16
+        
+        let leadingLine = UIView()
+        leadingLine.layer.borderColor = #colorLiteral(red: 0.94, green: 0.94, blue: 0.94, alpha: 1).cgColor
+        leadingLine.layer.borderWidth = 1
+        
+        let trailingLine = UIView()
+        trailingLine.layer.borderColor = #colorLiteral(red: 0.94, green: 0.94, blue: 0.94, alpha: 1).cgColor
+        trailingLine.layer.borderWidth = 1
+        
+        let label = UILabel()
+        label.attributedText = "Or".formattedText(
+            font: "Roboto-Light",
+            size: 15,
+            color: #colorLiteral(red: 0.52, green: 0.53, blue: 0.57, alpha: 1),
+            kern: 0
+        )
+        
+        stack.addArrangedSubview(leadingLine)
+        stack.addArrangedSubview(label)
+        stack.addArrangedSubview(trailingLine)
+        
+        leadingLine.widthAnchor.constraint(equalToConstant: 130).isActive = true
+        leadingLine.heightAnchor.constraint(equalToConstant: 2).isActive = true
+        trailingLine.widthAnchor.constraint(equalToConstant: 130).isActive = true
+        trailingLine.heightAnchor.constraint(equalToConstant: 2).isActive = true
```

Comment:
Normally, you activate constraints using:
```swift
NSLayoutConstraint.activate([...])
```

Why are you doing it different now?